common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -74,23 +74,38 @@ class @Problem
   renderProgressState: =>
     detail = @el.data('progress_detail')
     status = @el.data('progress_status')
+    graded = @el.data('graded')
 
     # Render 'x/y point(s)' if student has attempted question 
     if status != 'none' and detail? and (jQuery.type(detail) == "string") and detail.indexOf('/') > 0
         a = detail.split('/')
         earned = parseFloat(a[0])
         possible = parseFloat(a[1])
-        # This comment needs to be on one line to be properly scraped for the translators. Sry for length.
-        `// Translators: %(earned)s is the number of points earned. %(total)s is the total number of points (examples: 0/1, 1/1, 2/3, 5/10). The total number of points will always be at least 1. We pluralize based on the total number of points (example: 0/1 point; 1/2 points)`
-        progress_template = ngettext('(%(earned)s/%(possible)s point)', '(%(earned)s/%(possible)s points)', possible)
+
+        if graded == "True" and possible != 0
+            # This comment needs to be on one line to be properly scraped for the translators. Sry for length.
+            `// Translators: %(earned)s is the number of points earned. %(total)s is the total number of points (examples: 0/1, 1/1, 2/3, 5/10). The total number of points will always be at least 1. We pluralize based on the total number of points (example: 0/1 point; 1/2 points)`
+            progress_template = ngettext('%(earned)s/%(possible)s point earned (graded)', '%(earned)s/%(possible)s points earned (graded)', possible)
+        else
+            # This comment needs to be on one line to be properly scraped for the translators. Sry for length.
+            `// Translators: %(earned)s is the number of points earned. %(total)s is the total number of points (examples: 0/1, 1/1, 2/3, 5/10). The total number of points will always be at least 1. We pluralize based on the total number of points (example: 0/1 point; 1/2 points)`
+            progress_template = ngettext('%(earned)s/%(possible)s point earned (ungraded)', '%(earned)s/%(possible)s points earned (ungraded)', possible)
         progress = interpolate(progress_template, {'earned': earned, 'possible': possible}, true)
 
     # Render 'x point(s) possible' if student has not yet attempted question
-    if status == 'none' and detail? and (jQuery.type(detail) == "string") and detail.indexOf('/') > 0
-        a = detail.split('/')
-        possible = parseFloat(a[1])
-        `// Translators: %(num_points)s is the number of points possible (examples: 1, 3, 10). There will always be at least 1 point possible.`
-        progress_template = ngettext("(%(num_points)s point possible)", "(%(num_points)s points possible)", possible)
+    if status == 'none' or status == 0
+        if detail? and (jQuery.type(detail) == "string") and detail.indexOf('/') > 0
```

Comment:
@staubina @cahrens 

In progress.py, _to_js_status_str_ and _to_js_detail_str_ are responsible for the value of progress-detail and progress-status. 

If progress equals "none" (will only happen if max points possible is 0) then the value of progress-detail and progress-status are set to "0". Not "0/0" like we were expecting.

If progress is "0" (will happen if user attempts submission but gets it wrong) then progress-status is set to "none", and progress-detail is set to "0/max points possible".

Otherwise progress_status is either "in_progress" or "done", and progress_detail is some number out of some other number ("1/4"). 

So the case where detail is 0 that AJ was running into seems like it was because capa_base is returning "0" instead of "0/0", and that was converting to an integer instead of a string. I'm also not loving that a progress value of 0 -> a progress-status value of none, and a progress value of none -> a progress-status value of 0, but I'm concerned about changing that because I'm not sure what else depends on those values being that way.